### ISSUE TITLE
UserTags Button does not appear

### ISSUE THREAD

**Issue by tetralovania** [SrS2225a/BetterDiscord#18]
![image](https://user-images.githubusercontent.com/86831655/204096242-b5a00681-98da-48b0-b71a-cfccc2b19e8a.png)
As seen here (plugin is installed and activated, no errors), nothing shows up

**Comment by ServOKio** [comment]
Same problem

**Comment by SrS2225a** [comment]
Hello guys, terribly sorry about that. The reason why noting shows up is because when discord made the breaking change to their  internals, it pretty much broke all of the better discord plugins, including this one, and because of this I have decided to take a break from plugin development for a few months  as I was frustrated with discord for doing such a thing. 

But dont worry, I am back to plugin development now, and are fixing all my plugins as quick as I can! I will have the fix ready by tomorrow, which then will need to be accepted by better discord with the new changes which could take a few days.

Sorry about any inconvenience this may of caused

**Comment by SrS2225a** [comment]
Hi all,

I just wanted to let everyone here that I have made all the fixes necessary for this plugin to work once again. One thing to note though is since the QuickSwitcher is no longer "exposed" to plugins I had to get the react instance and patch it in a hacky way, which is why it took longer than my orginal estimate to fix than I orginally thought. The only downside to this method is that it needs to be force updated on the state node, which causes the QuickSwitcher to be a bit glitchy. I will leave this issue open for a few days in case anyone runs into anything else with the plugin
